packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -127,7 +127,7 @@ $search-transition-timing: 95ms;
 @mixin masthead {
   .#{$prefix}--masthead,
   :host(#{$dds-prefix}-masthead) {
-    position: fixed;
+    position: absolute;
```

Comment:
Just doublechecking this actually, we want the masthead to be fixed right?
cc: @kennylam

Reply:
@jeffchew It should be for the initial 25% of scrolling, then it would begin scrolling. If L1 is present, then that would then stick. I think for now this is more of a fix to allow the TOC to stick properly to the edge of the viewport, since at the moment it will go behind the masthead and not visible.

Reply:
Got it, so I'll approve this then!

Reply:
Do we have another issue to track updating the 25% scroll logic?
cc: @RobertaJHahn @kennylam @ariellalgilmore @larahanlon2

Reply:
> Do we have another issue to track updating the 25% scroll logic?
> cc: @RobertaJHahn @kennylam @ariellalgilmore @larahanlon2

@jeffchew I don't think so, I'll create one.